app/src/common/shared/com/igalia/wolvic/ui/widgets/WindowWidget.java

```diff
@@ -849,6 +849,12 @@ public void handleTouchEvent(MotionEvent aEvent) {
                 }
                 // Return to discard first click after focus
                 return;
```

Comment:
Here is where we focus the window but avoid passing the "down" event.

If we remove this `return;` line, the window gets the focus and then responds to the user's input, which feels more natural. What do you think?

Reply:
Yeah, this works too, and indeed a simpler approach.

I was thinking about removing this too, but a bit reluctant. Because this piece of code is used for all devices, and I just test with Quest 3. Not sure if removing this would have any unexpected effects on other devices :thinking: 

Anyway, I removed this in the last commit, and it worked with my Quest 3^^